config/install-latest-testing.yaml

```diff
@@ -51402,7 +51402,7 @@ spec:
       serviceAccountName: kyverno-cleanup-controller
       containers:
         - name: controller
-          image: "ghcr.io/kyverno/cleanup-controller:latest"
+          image: "kyverno/cleanup-controller:latest"
```

Comment:
Why is the default registry `ghcr.io` missing for the cleanup controller and the reports controller below?

Reply:
Sorry, I was sick. However, it’s fixed now.

Reply:
No worry, I hope you are feeling better now!